kotlin/openId4VP/src/main/java/io/mosip/openID4VP/common/Decoder.kt

```diff
@@ -0,0 +1,27 @@
+package io.mosip.openID4VP.common
+
+import io.mosip.openID4VP.authorizationRequest.exception.AuthorizationRequestExceptions
+import org.apache.commons.codec.binary.Base64
+import java.nio.charset.StandardCharsets
+
+object Decoder {
+    private val logTag = Logger.getLogTag(this::class.simpleName!!)
```

Comment:
this logger is initialised here, how will it have traceabilityId.

Reply:
In authenticate Verifier method of OpenId4VP file i am calling Logger.setTraceability(traceabilityId) method to set the id and this id will be used throughout the project.